src/Hocon.API.Tests/HoconAPISpec.ApproveConfiguration.approved.txt

```diff
@@ -7,30 +7,29 @@ namespace Hocon
         protected CDataConfigurationElement() { }
         protected override void DeserializeElement(System.Xml.XmlReader reader, bool serializeCollectionKey) { }
     }
-    public class Config : Hocon.HoconRoot, System.IEquatable<Hocon.Config>, System.Runtime.Serialization.ISerializable
+    public class Config : Hocon.HoconObject, System.IEquatable<Hocon.Config>, System.Runtime.Serialization.ISerializable
     {
-        [System.ObsoleteAttribute("For json serialization/deserialization only", true)]
-        protected Config() { }
-        protected Config(Hocon.HoconValue value) { }
-        protected Config(Hocon.HoconValue value, Hocon.Config fallback) { }
-        public Config(Hocon.HoconRoot root) { }
-        public Config(Hocon.HoconRoot root, Hocon.Config fallback) { }
+        public Config(Hocon.HoconElement root) { }
         [System.ObsoleteAttribute("Used for serialization only", true)]
-        public Config(System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context) { }
+        protected Config(System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context) { }
         public static Hocon.Config Empty { get; }
-        public virtual System.Collections.Generic.IReadOnlyList<Hocon.HoconValue> Fallbacks { get; }
+        public virtual System.Collections.Generic.IReadOnlyList<Hocon.HoconObject> Fallbacks { get; }
```

Comment:
What's the significance of this change?

Reply:
There's no such thing as `HoconValue` anymore, the structure is simplified.